modules/consensus/consensusset.go

```diff
@@ -211,6 +211,9 @@ func (cs *ConsensusSet) Close() error {
 
 // CurrentBlock returns the latest block in the heaviest known blockchain.
 func (cs *ConsensusSet) CurrentBlock() (block types.Block) {
+	cs.mu.RLock()
+	defer cs.mu.RUnlock()
```

Comment:
do you need this on the other exported functions as well? I assume it's protecting `cs.db`